build/release.mk

```diff
@@ -60,10 +95,10 @@ ifndef SNAPSHOT
 	$(error SNAPSHOT is undefined)
 endif
 
-# snapshot replaces version tag for local builds
+# snapshot replaces version tag for local builds, also --skip-validate to avoid git errors
 SNAPSHOT := ${SNAPSHOT}
 ifeq ($(SNAPSHOT), true)
```

Comment:
Not sure if this actually works, I've tried once locally and did not

Reply:
it worked for me :)